### ISSUE TITLE
Can't get B&N key

### ISSUE THREAD

**Issue by akacat63** [apprenticeharper/DeDRM_tools#2038]
I know I'm filling in the username and password correctly (I've logged in and out of bn.com a couple times using the exact same credentials), and I have internet access with no proxies. I keep getting the error message "Could not retrieve key. Check username, password and internet connectivity and try again."

Is this broken? Is there a workaround? Is there a way I can run just it from the command line to get more detailed error messages?

**Comment by ElleKayEm** [comment]
Yes, this version's key retrieval broke.  Try noDRM's DeDRM:  https://github.com/noDRM/DeDRM_tools
